<dec f='qv2ray/test/catch.hpp' l='2547' type='Catch::AssertionInfo'/>
<use f='qv2ray/test/catch.hpp' l='2560' u='r' c='_ZN5Catch16AssertionHandlerD1Ev'/>
<use f='qv2ray/test/catch.hpp' l='8216' u='w' c='_ZN5Catch16AssertionHandlerC1ERKNS_9StringRefERKNS_14SourceLineInfoES1_NS_17ResultDisposition5FlagsE'/>
<use f='qv2ray/test/catch.hpp' l='8221' u='r' c='_ZN5Catch16AssertionHandler10handleExprERKNS_20ITransientExpressionE'/>
<use f='qv2ray/test/catch.hpp' l='8224' u='r' c='_ZN5Catch16AssertionHandler13handleMessageENS_9ResultWas6OfTypeERKNS_9StringRefE'/>
<use f='qv2ray/test/catch.hpp' l='8254' u='r' c='_ZN5Catch16AssertionHandler33handleUnexpectedInflightExceptionEv'/>
<use f='qv2ray/test/catch.hpp' l='8258' u='r' c='_ZN5Catch16AssertionHandler31handleExceptionThrownAsExpectedEv'/>
<use f='qv2ray/test/catch.hpp' l='8261' u='r' c='_ZN5Catch16AssertionHandler34handleExceptionNotThrownAsExpectedEv'/>
<use f='qv2ray/test/catch.hpp' l='8265' u='r' c='_ZN5Catch16AssertionHandler34handleUnexpectedExceptionNotThrownEv'/>
<use f='qv2ray/test/catch.hpp' l='8269' u='r' c='_ZN5Catch16AssertionHandler25handleThrowingCallSkippedEv'/>
<offset>0</offset>
